pydm/widgets/archiver_time_plot.py

```diff
@@ -926,6 +942,8 @@ def addYChannel(
             useArchiveData=useArchiveData,
             liveData=liveData,
         )
+        self.requestDataFromArchiver()
```

Comment:
If however it's still useful to make this call here, you'll want to put an `if not is_qt_designer()` check (imported from `pydm.utilities`) around it so it doesn't make failing archive calls when adding curves to plots in Designer. 

When I was trying this without the change I mention above, the signal was sent and handled before the plot fully rendered leading to no initial data fetch.

Reply:
Added in this check.
I was also getting errors caused by `self._min_x` being initialized as `None`.
Fixed this by setting it to `self._starting_timestamp - DEFAULT_TIME_SPAN` in the `__init__` since it's set in `self.updateXAxis()` after startup.